spec/features/create_todo_items_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe "Todo items" do
+   before do
+      @user = create(:user)
+      visit(user_path(@user))
+    end
+  
+  describe "Creating todo items" do
+   
+    it "allows user to create todo item" do
+      expect{ @user.items.create(title: 'new task') }.to change{ @user.items.count }.by(1) 
```

Comment:
This isn't really a feature test.

In your features, you want to drive all interaction via the browser actions, like `click_link`, `fill_in`, and then verify the page changes in the way you expect.